migrations/20230311081917_envelopes_table.sql

```diff
@@ -1,5 +1,6 @@
 CREATE TABLE IF NOT EXISTS envelopes (
   id              INTEGER PRIMARY KEY AUTOINCREMENT,
+  received_at     INTEGER,
```

Comment:
SQLite's docs on [date and time datatypes](https://www.sqlite.org/datatype3.html#date_and_time_datatype) mention that the integer is expected to represent the number of seconds. This now represents milliseconds. We either change the format in the database to represent milliseconds or if we go with a bit more dirty solution I'd say to rename the column to `received_at_ms`. What do you think?

Reply:
As the docs state the SQLite does not have notion of datetime types in it. 
> SQLite does not have a storage class set aside for storing dates and/or times.
Meaning users basically free to use whatever they want. 

In our case the `INTEGER` is an `INTEGER` nothing else, and nothing more, which we happened to use it for storing the timestamp in the form of `i64`, which we then get back. Database in this case does not care what the number in there  and what it represents. 

Using milliseconds gives us better precision and I do not see the point to put that in the name of the field, which feels very wrong.

Reply:
Yeah that makes sense 👍